.github/workflows/perl-critic.yml

```diff
@@ -5,7 +5,7 @@ on:
   pull_request:
   push:
     branches:
-      - 'master'
+      - '*'
```

Comment:
Instead of

```
on:
  pull_request:
  push:
    branches:
      - '*'
```

make that simply:

```
on: [push, pull_request]
```

That's also what we have in https://github.com/os-autoinst/os-autoinst/blob/master/.github/workflows/ci.yml#L4

Reply:
done